actors/builtin/miner/miner_actor.go

```diff
@@ -1608,11 +1613,14 @@ func processEarlyTerminations(rt Runtime) (more bool) {
 		// Unlock funds for penalties.
 		// TODO: handle bankrupt miner: https://github.com/filecoin-project/specs-actors/issues/627
 		// We're intentionally reducing the penalty paid to what we have.
-		penalty, err = st.UnlockUnvestedFunds(store, rt.CurrEpoch(), penalty)
+		available := st.GetUnlockedBalance(rt.CurrentBalance())
```

Comment:
Blocking: `available` -> `unlocked`, or change parameters for `PenalizeFundsInPriorityOrder`